frontend/src/components/edition/editionOverview.tsx

```diff
@@ -71,20 +71,24 @@ export function EditionOverview({ editionId }: EditionOverviewProps) {
     }
 
     async function handleSaveName() {
-        setEditName(false);
-        if (edition) {
-            const response: AxiosResponse = await saveEditionName(edition._links.self.href, name);
-            if (response.status === StatusCodes.OK) {
-                if (response.data) {
-                    await Promise.all([
-                        mutate(apiPaths.editions),
-                        mutate(edition._links.self.href, response.data),
-                    ]);
+        if (name === "") {
+            alert(capitalize(t("input empty")));
+        } else {
```

Comment:
An input field can have a required value, right?

Reply:
Oh I see you added that further on. Is this check then still needed?

Reply:
Yeah, I added it there to test it but that doesn't work as the confirm knop is not the same as with a form. Here the confirm knop is used to trigger a function which takes the state. The required field doesn't work here because of that as both are not linked directly.

Reply:
Okay!